mutex_legacy.go

```diff
@@ -1,3 +1,6 @@
+//go:build !go1.24
+// +build !go1.24
```

Comment:
Btw does this mean "not 1.24" or "not <1.24"?

Reply:
It is effectively "not <1.24" as the compiler uses all of the tags from the previous versions. I don't have a good source on this but that's the behavior I have observed.